app/controllers/flash_teams_controller.rb

```diff
@@ -643,111 +647,111 @@ def event_search
     render :partial => "event_search_results"
 
    end
-   
+
    def task_portal
-   
+
    		@id_team = params[:id]
-   		
+
    		#@flash_team = FlashTeam.find(params[:id])
-   		
-   		if valid_user?	
+
+   		if valid_user?
```

Comment:
Maybe take a look at your editor settings that are removing this whitespace, and try to turn it off? Would be alright to leave these whitespaces in there, and it would make for a more compact and easy-to-review PR.